Makefile

```diff
@@ -634,11 +644,20 @@ endef
 		$(call go-get,github.com/cpuguy83/go-md2man); \
 	fi
 
+# $BUILD_TAGS variable is used in hack/golangci-lint.sh
 .PHONY: varlink_generate
+ifneq (or $(findstring varlink,$(BUILDTAGS)),$(findstring varlink,$(BUILD_TAGS)))
```

Comment:
I'm super rusty on Makefile magic, but this doesn't look right to me. Are you looking for `varlink` in either of those two variables? If so would this work?
```makefile
ifneq (,$(findstring varlink,$(BUILDTAGS) $(BUILD_TAGS)))
```

Reply:
yes, I'm looking for varlink in those two. I'll give that a try, thanks!

Reply:
there seems to be some integration test issues that I'm looking into as well.

Reply:
`make lint` fails with:
```
# $BUILD_TAGS variable is used in hack/golangci-lint.sh
 .PHONY: varlink_generate
-ifneq (or $(findstring varlink,$(BUILDTAGS)),$(findstring varlink,$(BUILD_TAGS)))
+ifneq (,$(findstring varlink,$(BUILD_TAGS) $(BUILDTAGS)))
 varlink_generate: .gopathok pkg/varlink/iopodman.go ## Generate varlink
 else
 varlink_generate:
```

Reply:
Probably some magic syntax detail I'm missing.

Well, does the `or` format work? If you run `make` with varlink in either variable, does it trigger the desired varlink_generate?

Reply:
The `or` format works well for `make lint`. I don't need to specify `varlink` buildtag for tunnel and abi as it will automatically use it from the golangci-lint.sh file. Both `make` and `make BUILDTAGS='varlink' work correctly as well.

Reply:
OK - sorry to have muddled things. LGTM, thanks.

Reply:
btw, what you suggested *I think* should work as well, which is what seems to be suggested in the [official make doc](https://www.gnu.org/software/make/manual/make.html#Testing-Flags), but for whatever reason it doesn't seem to work in this case. Maybe more eyes would help..